async/retry.ts

```diff
@@ -74,10 +74,8 @@ const defaultRetryOptions: Required<RetryOptions> = {
  * Retries as long as the given function throws. If the attempts are exhausted,
  * throws a {@linkcode RetryError} with `cause` set to the inner exception.
  *
- * The backoff is calculated by multiplying `minTimeout` with `multiplier` to
- * the power of the current attempt counter (starting at 0 up to `maxAttempts
- * - 1`). It is capped at `maxTimeout` however. How long the actual delay is,
- * depends on `jitter`.
+ * The backoff is calculated by multiplying `minTimeout` with `multiplier` to the power of the current attempt counter (starting at 0 up to `maxAttempts - 1`). It is capped at `maxTimeout` however.
+ * How long the actual delay is, depends on `jitter`.
```

Comment:
I fixes based on this. https://github.com/denoland/deno_std/blob/e0ef24091e87f84d44d495d432d611625b281249/async/retry.ts#L41